crates/l2/sequencer/l1_committer.rs

```diff
@@ -159,7 +162,11 @@ impl Committer {
             &account_updates,
         )?;
 
-        let blobs_bundle = self.generate_blobs_bundle(&state_diff)?;
+        let blobs_bundle = if !self.validium {
+            self.generate_blobs_bundle(&state_diff)?
```

Comment:
The call to `prepare_state_diff` can be moved inside the `if` case since there's no point in computing said state diff in validium mode (at least for now)